src/pagination/pagination.ts

```diff
@@ -33,22 +33,32 @@ import {NgbPaginationConfig} from './pagination-config';
 
         <li *ngIf="directionLinks" class="page-item" [class.disabled]="!hasNext()">
           <a aria-label="Next" class="page-link" href (click)="!!selectPage(page+1)">
-            <span aria-hidden="true">&raquo;</span>
-            <span class="sr-only">Next</span>
+            <span aria-hidden="true">{{nextText}}</span>
+            <span class="sr-only">{{_nextTextSr}}</span>
           </a>
         </li>
         
         <li *ngIf="boundaryLinks" class="page-item" [class.disabled]="!hasNext()">
           <a aria-label="Last" class="page-link" href (click)="!!selectPage(pageCount)">
-            <span aria-hidden="true">&raquo;&raquo;</span>
-            <span class="sr-only">Last</span>
+            <span aria-hidden="true">{{lastText}}</span>
+            <span class="sr-only">{{_lastTextSr}}</span>
           </a>                
         </li>        
       </ul>
     </nav>
   `
 })
 export class NgbPagination implements OnChanges {
+  private _firstText;
```

Comment:
I don't think you should use private - these are all to be accessed by the template, it doesn't seem like a private property.